templates/uimodules/image.html

```diff
@@ -18,7 +18,7 @@ <h1 class="image-edit-title">{{escape(sharedfile.get_title())}}</h1>
         {% if show_attribution_in_title %}
             <div class="image-poster">
                 <a href="{{attribution['path']}}" title="{{escape(attribution['name'])}}">
-                    <img class="avatar--img" src="{{attribution['thumbnail_url']}}" width="48" height="48" alt="">
+                    <img class="avatar--img" src="{{attribution['thumbnail_url']}}" width="48" height="48" alt="{{escape(attribution['name'])}}">
```

Comment:
Same as above.

Reply:
And this is the only one of the three that escapes the name value. Should I be doing that for the others?

Reply:
`name` here could either be the user `name` property (which is safe to not escape) or the 'display name' of the Shake, which could be the user-supplied title of the shake, and that content should be escaped since it isn't restricted to certain characters.